payments/base/arch/src/main/java/com/appcoins/payments/arch/error/ServerUnreachableException.kt

```diff
@@ -0,0 +1,3 @@
+package com.appcoins.payments.arch.error
+
+class ServerUnreachableException : Throwable()
```

Comment:
It looks like just a replacement for `java.net.UnknownHostException` without any added features.
What if we just use `java.net.UnknownHostException` directly?